dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -301,7 +301,18 @@ CWWKD1026.ignore.case.not.text.useraction=Ensure the sort criteria is applied \
  to the correct entity attribute and remove the instruction to ignore case if the \
  entity attribute does not represent text.
 
-# TODO CWWKD1027 removed prior to GA of data-1.0 and can be reused
+CWWKD1027.datastore.lacks.java.comp=CWWKD1078E: One or more repository interfaces \
+ ({0}) from the {1} application artifact specify a {2} data store value that \
```

Comment:
```suggestion
 ({0}) from the {1} application artifact specify a {2} dataStore value that \
```
Suggestion: match the attribute name to make it easier for a user to find, however if the other messages all use `data store` to reference this attribute then I'm fine with it.